src/process/validation/rules/validateMask.ts

```diff
@@ -90,8 +98,8 @@ export function matchInputMask(value: any, inputMask: any) {
 }
 
 export const shouldValidate = (context: ValidationContext) => {
-    const { component, value } = context;
-    if (!isValidatableComponent(component) || !value) {
+    const { component, value, instance } = context;
+    if (!isValidatableComponent(component) || !value || shouldSkipMaskValidation(context)) {
```

Comment:
This is something of a nit, but wouldn't the check for skipMaskValidation just be better inside of the isValidatableComponent fn?

Reply:
e.g. the component is not validatable if it has skipMaskValidation

Reply:
I refactored the `isValidatableComponent` function to improve readability. The previous implementation with two additional conditions made the function difficult to understand at first look. Is it fine?